### ISSUE TITLE
Lots of unit test failures on my machine

### ISSUE THREAD

**Issue by luser** [gf3/IRC-js#65]
Ubuntu 12.04 x86-64, node v0.10.12, lots of unit tests fail:
- test-irc:
  18 passing (348ms)
  3 pending
  6 failing
- test-logger: all passing
- test-objects:
  16 passing (101ms)
  5 failing
- test-parser: all passing

**Comment by ghost** [comment]
Yeah, same here. They broke after some node update, I’ve been too lazy to fix them.
